modules/site-health/admin-site-health.php

```diff
@@ -168,6 +168,15 @@ public function info_options( $debug_info ) {
 				}
 			}
 		}
+
+		// does a wpml-config.xml exists ?
+		$wpml = new PLL_WPML_Config();
+		$wpml_files = $wpml->files;
```

Comment:
```suggestion
		$wpml_files = PLL_WPML_Config::instance()->get_files();
```
This will be ok if you follow my suggestion in the other comment.